src/engraving/rendering/dev/horizontalspacing.cpp

```diff
@@ -241,7 +241,7 @@ void HorizontalSpacing::spaceRightAlignedSegments(Measure* m, double segmentShap
         }
     }
     // Compute spacing
-    static constexpr double arbitraryLowReal = -10000.0;
+    static constexpr double arbitraryLowReal = -MAX_DOUBLE;

```

Comment:
In this case too you might want to replace `arbitraryLowReal` with `DBL_MAX` directly in all places